src/frontend/src/lib/i18n/en.json

```diff
@@ -726,5 +726,12 @@
 				"cannot_call": "The call cannot be processed"
 			}
 		}
+	},
+	"carousel": {
+		"text": {
+			"next_slide": "Next slide",
```

Comment:
Those keys will be used as `aria-label` for the respective control/indicator buttons.